### DISCUSSION TITLE
error in uvc camera

### DISCUSSION THREAD

**Discussion by martten20** [stella-cv/stella_vslam#208] [Q&A] [answered]
hi there,
i want to use kinect 360 in openvslam but it won't run and gives me the "can't open camera 1" error and the camera is number 1 in my device.
 so is the kinect supported in the openvslam or not and if not is there any solution to run the kinect in openvslam?
thanks in advance.

**Comment by MhmdAdnan** [comment]
+111111

**Comment by MhmdAdnan** [comment]
anyone cane help as soon as possible

**Comment by ymd-stella** [comment, marked as answer]
Use openvslam_ros and the ROS node for the camera.
